Source/Private/ASDisplayNodeInternal.h

```diff
@@ -204,9 +204,9 @@ FOUNDATION_EXPORT NSString * const ASRenderingEngineDidDisplayNodesScheduledBefo
   NSMutableArray<ASDisplayNode *> *_yogaChildren;
 #endif
 #if YOGA_TREE_CONTIGUOUS
-  YGNodeRef _yogaNode;
   __weak ASDisplayNode *_yogaParent;
   ASLayout *_yogaCalculatedLayout;
+  BOOL _yogaLayoutInProgress;
```

Comment:
Yes, ideally this would be both thread-affined and node-affined.

Considering the other PR doing this with ASLayoutElementContext, I wonder if instead we should have a table inside each ASDisplayNode with an unordered_map from pthread ID to an object context?